static/CACHE/css/base.2ff388b33c76.css

```diff
@@ -0,0 +1,456 @@
+/* Global Variables */
```

Comment:
We are not tracking any of these `CACHE` files. Please pull the latest changes from develop and delete this folder.